lib/pages/authentication-page.dart

```diff
@@ -20,9 +21,12 @@ class AuthenticationPageState extends State<AuthenticationPage> {
   final _passwordController = new TextEditingController();
   String _email;
   String _password;
+  String skip;
+
+  var _userService = GetIt.I.get<UserService>();
+  var _authenticationService = GetIt.I.get<AuthenticationService>();
 
   final _authService = GetIt.I.get<AuthenticationService>();
```

Comment:
duplicate service. use one of _authenticationService or _authService but not both. Remove one of them